packages/redis_client/lib/src/redis_client.dart

```diff
@@ -287,11 +399,23 @@ class RedisJson {
   /// Equivalent to the `JSON.DEL` command.
   /// https://redis.io/commands/json.del
   Future<void> delete({required String key}) {
-    return _client.sendCommand(['JSON.DEL', key, r'$']);
+    return _client.execute(['JSON.DEL', key, r'$']);
   }
 }
 
 extension on RedisSocketOptions {
   /// The connection URI for the Redis server derived from the socket options.
   Uri get connectionUri => Uri.parse('redis://$host:$port');
 }
+
+final class _NoopRedisLogger implements RedisLogger {
```

Comment:
Just to make sure I'm not missing anything: this is the only `RedisLogger` implementation at the moment?

Reply:
It's the only one that ships with the package but we'll create our own.